XXX push upstream

../gthumb-3.12.8/extensions/image_rotation/rotation-utils.c:229:32: error: use of undeclared identifier 'true'
  229 |                 g_task_return_boolean (task, true);
      |                                              ^
../gthumb-3.12.8/extensions/image_rotation/rotation-utils.c:247:32: error: use of undeclared identifier 'true'
  247 |                 g_task_return_boolean (task, true);
      | 

Index: extensions/image_rotation/rotation-utils.c
--- extensions/image_rotation/rotation-utils.c.orig
+++ extensions/image_rotation/rotation-utils.c
@@ -21,6 +21,7 @@
 
 #include <config.h>
 #include <unistd.h>
+#include <stdbool.h>
 #include <sys/types.h>
 #include <glib/gi18n.h>
 #include <gio/gio.h>
